app/main/views/questionnaire.py

```diff
@@ -46,12 +46,18 @@ def survey(eq_id, collection_id, location):
 
         context = questionnaire_manager.get_rendering_context()
         template = questionnaire_manager.get_rendering_template()
-
-        return render_template(template,
-                               meta=context['meta'],
-                               content=context['content'],
-                               navigation=context['navigation']
-                               )
+        try:
+            theme = context['meta']['survey']['theme']
+            logger.info("Theme selected: {} ".format(theme))
+        except KeyError:
```

Comment:
The scheme parser would fall over before here if theme was missing, so this might be redundant